portfolio/src/main/webapp/script.js

```diff
@@ -20,7 +20,22 @@ function off() {
 }
 // asynchronously fetch content from server
 async function getContent() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('content-container').innerText = quote;
+    const response = await fetch('/data');
+    const hardCodedContent = await response.json();
+  
+    const contentListElement = document.getElementById("content-container");
+    contentListElement.innerHTML = '';
+    contentListElement.appendChild(
+        createListElement("Element 1: " + hardCodedContent[0]));
+    contentListElement.appendChild(
+        createListElement("Element 1: " + hardCodedContent[1]));
+    contentListElement.appendChild(
+        createListElement("Element 1: " + hardCodedContent[2]));
```

Comment:
Element 2:

Reply:
typos also addressed in merge pull request #8 [e097599](https://github.com/charmychancey/step-my-portfolio/pull/6/commits/e097599814d93a03893c03d9b1699464f266e35c)